app/cryptography/token_helper.py

```diff
@@ -8,19 +9,17 @@
 
 from app.authentication.invalid_token_exception import InvalidTokenException
 
+logger = get_logger()
 
-def decrypt_jwe(encrypted_token, secret_store, purpose, default_kid=None):
+
+def decrypt_jwe(encrypted_token, secret_store, purpose):
     try:
         jwe_token = jwe.JWE(algs=['RSA-OAEP', 'A256GCM'])
         jwe_token.deserialize(encrypted_token)
 
-        try:
-            jwe_kid = extract_kid_from_header(encrypted_token)
-        except InvalidTokenException as e:
-            if default_kid:
-                jwe_kid = default_kid
-            else:
-                raise e
+        jwe_kid = extract_kid_from_header(encrypted_token)
+
+        logger.info("Decrypting JWE", KID=jwe_kid)
```

Comment:
From the logging standards at https://digitaleq.atlassian.net/wiki/display/SDC/Logging, the keys should be lower case - KID -> kid